backend/indexer/indexer.py

```diff
@@ -42,17 +44,19 @@ def get_data_point_fqn_to_hash_map(
     return data_point_fqn_to_hash
 
 
-async def sync_data_source_to_collection(inputs: DataIngestionConfig):
+def sync_data_source_to_collection(inputs: DataIngestionConfig):
```

Comment:
Should not change this

Reply:
Even if I use background task of fastAPI, it just awaits async functions (see [this](https://github.com/encode/starlette/blob/master/starlette/background.py#L25)), which will block fastapi's event loop

Reply:
Let's please revert changes to this function